lib/charm.ts

```diff
@@ -0,0 +1,90 @@
+import StatTypes from "../public/data/stat-types.json";
+
+export enum CharmPosition {
+  Left,
+  Mid,
+  Right,
+}
+
+export interface StatSummary {
+  name: string;
+  value: string;
+}
+
+export class Charm {
```

Comment:
any reason you are using a `class` instead of a `type`? Seems like there isn't anything here that requires the `class` at all.